src/main/git-elegant

```diff
@@ -102,8 +102,10 @@ push-after-rebase() {
 
 feature() {
     _validate "$1" "Feature name"
+    git stash
```

Comment:
@ivan-podorozhnyi Seems that here and on `git stash pop` we need some checks, because command `feature` might be used with no changes on branch, so `git stash` won't create stash and `git stash pop` will apply some unsuitable stash.

Reply:
@AntonKarabaza Ok, I will add some checks.